bin/audit-svg-colors.js

```diff
@@ -293,12 +324,19 @@ function printReplacementRules( replacementObjects ) {
 }
 
 function formatReplacementRules( rules ) {
-	return _.sortBy( rules, 'to.name' ).map( ( rule ) => {
-		const valueFrom = rule.from.value.padEnd( 7 );
-		const valueTo = rule.to.value.padEnd( 7 );
-
-		return `${ valueFrom } → ${ valueTo } (${ rule.to.name })`;
-	} );
+	if ( rules && rules.length ) {
```

Comment:
Previously, this function would always return an array. Now if `rules` is an empty array or not an array, it will return `undefined`. Sounds like we should take care of this.

Reply:
I have added the else block and returned an empty array.